### ISSUE TITLE
Package with libpaper

### ISSUE THREAD

**Issue by rrthomas** [rrthomas/psutils#70]
It would be nice if users didn't have to install `libpaper` separately (see #69).

Packaging with `pip` seems possible, with https://github.com/pypa/cibuildwheel/

But this doesn't give much help for building and installing `libpaper`—it's more aimed at building Python extensions.

An alternative would be to package libpaper and PSUtils for Conda, but that would only help Conda users.

**Comment by cbm755** [comment]
Drive-by comment, without much thought behind it...  but would the port to PyMuPDF perhaps make `libpaper`  redundant?  See `pymupdf.paper_sizes()` and `pymupdf.paper_size()`.

**Comment by rrthomas** [comment]
> Drive-by comment, without much thought behind it... but would the port to PyMuPDF perhaps make `libpaper` redundant? See `pymupdf.paper_sizes()` and `pymupdf.paper_size()`.

Good idea, but MuPDF doesn't seem to have any sort of configurability: looking at the source it just has a small number of hard-wired paper sizes. Given that Ghostscript itself can use libpaper, it would make sense if MuPDF could too; but that's not a PSUtils issue!
